src/frontend/components/Dashboard.jsx

```diff
@@ -22,7 +22,7 @@ function UserTabs(props) {
   const { role, inputs, handleSettings } = props;
   if (role === 'admins' || role === 'editors') {
     return (
-      <Tabs defaultActiveKey="form">
+      <Tabs defaultActiveKey="map">
```

Comment:
Sorry, I will change this back at some point but since I'm working on the map this just saves me a ton of time